packages/util_linux.rb

```diff
@@ -0,0 +1,22 @@
+require 'package'
+
+class Util_linux < Package
+  description 'essential linux tools'
+  homepage 'https://www.kernel.org'
+  version '2.30'
+  source_url 'https://www.kernel.org/pub/linux/utils/util-linux/v2.30/util-linux-2.30.tar.xz'
+  source_sha256 'c208a4ff6906cb7f57940aa5bc3a6eed146e50a7cc0a092f52ef2ab65057a08d'
+
+  depends_on 'python3'
+  depends_on 'vdev'
```

Comment:
Need dependency to `libcap_ag`?

When I compile this, this crerates following files.  It doesn't use python3 if python27 is installed.  You may need to specify python path at configure option to force to use python3.  Also bash_completion stuff conflicts with bash_completion.rb.  I don't care about latter.  It's just FYI.
```
drwxr-xr-x chronos/chronos       0 2017-09-18 03:34 usr/local/lib/python2.7/
drwxr-xr-x chronos/chronos       0 2017-09-18 03:34 usr/local/lib/python2.7/site-packages/
drwxr-xr-x chronos/chronos       0 2017-09-18 03:34 usr/local/lib/python2.7/site-packages/libmount/
-rwxr-xr-x chronos/chronos  255832 2017-09-18 03:34 usr/local/lib/python2.7/site-packages/libmount/pylibmount.so
-rw-r--r-- chronos/chronos      27 2017-09-18 03:34 usr/local/lib/python2.7/site-packages/libmount/__init__.py
-rw-r--r-- chronos/chronos     187 2017-09-18 03:34 usr/local/lib/python2.7/site-packages/libmount/__init__.pyo
-rw-r--r-- chronos/chronos     187 2017-09-18 03:34 usr/local/lib/python2.7/site-packages/libmount/__init__.pyc
-rwxr-xr-x chronos/chronos    1063 2017-09-18 03:34 usr/local/lib/python2.7/site-packages/libmount/pylibmount.la
...
-rw-r--r-- chronos/chronos    2591 2017-09-18 03:34 usr/local/share/bash-completion/completions/setterm
-rw-r--r-- chronos/chronos    1249 2017-09-18 03:34 usr/local/share/bash-completion/completions/zramctl
-rw-r--r-- chronos/chronos    1304 2017-09-18 03:34 usr/local/share/bash-completion/completions/lsipc
-rw-r--r-- chronos/chronos     576 2017-09-18 03:34 usr/local/share/bash-completion/completions/ipcmk
-rw-r--r-- chronos/chronos     509 2017-09-18 03:34 usr/local/share/bash-completion/completions/colrm
-rw-r--r-- chronos/chronos     726 2017-09-18 03:34 usr/local/share/bash-completion/completions/blockdev
-rw-r--r-- chronos/chronos    1167 2017-09-18 03:34 usr/local/share/bash-completion/completions/lsns
-rw-r--r-- chronos/chronos     721 2017-09-18 03:34 usr/local/share/bash-completion/completions/fallocate
-rw-r--r-- chronos/chronos    1007 2017-09-18 03:34 usr/local/share/bash-completion/completions/hexdump
-rw-r--r-- chronos/chronos    1961 2017-09-18 03:34 usr/local/share/bash-completion/completions/lsblk
-rw-r--r-- chronos/chronos     874 2017-09-18 03:34 usr/local/share/bash-completion/completions/flock
...
```

Reply:
Changed to python27.
I don't think that I can do something about the completion stuff.